### PR TITLE
feat: use fetched notification_email instead of stored email

### PR DESCRIPTION
**Pull request by WikiRik**
See https://github.com/AEGEE/frontend/pull/2057

### CODE REVIEW

File: lib/positions.js [hunk 1]
```diff
@@ -89,7 +114,7 @@ exports.listPositionsWithApprovedCandidates = async (req, res) => {
             .filter((candidate) => candidate.status !== 'rejected')
             .map((candidate) => {
                 if (candidate.status === 'approved') {
-                    return helpers.blacklistObject(candidate.toJSON(), ['email']); // the email should be visible to JC only
+                    return helpers.blacklistObject(candidate.toJSON(), ['email', 'notification_email']); // the email should be visible to JC only
```
Comment (LeonVreling) [inline lib/positions.js:117]: Why do you use both the old and the new everywhere updated notificiation email here?

Comment (WikiRik) [reply lib/positions.js:117]: Older applications still use the email stored in the database, I want that neither are displayed to others

Comment (LeonVreling) [reply lib/positions.js:117]: Okay sounds good

**Review by LeonVreling — APPROVED** [review]

**Pull request merged**